README.md

```diff
@@ -117,11 +117,16 @@ docker run -d --name influxdb \
       --network nuodb-net \
       -p 8086:8086 \
       -p 8082:8082 \
+      --env DOCKER_INFLUXDB_INIT_MODE= setup \
+      --env DOCKER_INFLUXDB_INIT_USERNAME= <your_username> \
+      --env DOCKER_INFLUXDB_INIT_PASSWORD= <your_password> \
+      --env DOCKER_INFLUXDB_INIT_ORG= <name_of_organization> \
+      --env DOCKER_INFLUXDB_INIT_RETENTION= <bucket_retention_time> \
+      --env DOCKER_INFLUXDB_INIT_BUCKET= <influxdb_bucket_name> \
+      --env DOCKER_INFLUXDB_INIT_ADMIN_TOKEN= <your_influxdb_token> \
```

Comment:
There is an extra space in all of these, e.g. `DOCKER_INFLUXDB_INIT_MODE= setup`. This should be `DOCKER_INFLUXDB_INIT_MODE=setup` (no space), so that it parses as one argument. Can you remove each of these spaces?